### DISCUSSION TITLE
Component is positioned centered veritically?

### DISCUSSION THREAD

**Discussion by manulsan** [graphieros/vue-data-ui#41] [Q&A] [answered]
Dear

Is component positioned centered in your client rectangle area?
I could not find way for this one to make it vertically centered.

Please refer below picture,  top-padding and bottom padding seems to be different.

![image](https://github.com/graphieros/vue-data-ui/assets/31757599/42b8e2ea-04c9-4505-891e-d86f7a0b7263)

 if these component has style option with "padding, margin", it would be better

Thanks you.

**Comment by graphieros** [comment, marked as answer]
Hi @manulsan 

Chart component are currently designed to work with titles.
When a title has content, the chart is centered as you would expect.

I will consider making changes to add config options to add padding on the container, however, you can already obtain the result you want by targeting the css class of the chart container and add padding top. For example:

```css
.vue-ui-gauge,
.vue-ui-wheel,
.vue-ui-thermometer {
  padding-top: 24px;
}
```
You may need to apply this style globally, not in a style scoped context.
